.github/_typos.toml

```diff
@@ -24,7 +24,9 @@ extend-exclude = [
     "PopulationByAdmin1.csv",
     "WomensSuffrage.txt",
     "SK-dotnet.sln.DotSettings",
-    "**/azure_ai_search_hotel_samples/README.md"
+    "**/azure_ai_search_hotel_samples/README.md",
+	"**/Demos/ProcessFrameworkWithAspire/ProcessFramework.Aspire/ProcessFramework.Aspire.ProcessOrchestrator/Program.cs",
+	"**/Demos/ProcessFrameworkWithAspire/**/*.http"
```

Comment:
nit: indentation

Reply:
As soon as I save the file it auto-indents with a new line. Guess it's a repo rule...

Reply:
/fixed

Reply:
@crickman tabs vs spaces ;-)